easygraph/functions/graph_embedding/sdne.py

```diff
@@ -18,18 +18,18 @@ def get_relation_of_index_and_node(graph):
     return idx2node, node2idx
 
 
-def l_2nd(beta):
+def l_2and(beta):
     from tensorflow.python.keras import backend as K
 
-    def loss_2nd(y_true, y_pred):
```

Comment:
I think the "2nd" means second?

Reply:
Yes you're right. The typos fixed are made automatically and I didn't review them all...

I'll double check the changes. :)